libs/design-system/search-autocomplete/src/lib/search-autocomplete.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="search-autocomplete-container" [class.sticky]="sticky()">
+  <mat-form-field class="search-field" subscriptSizing="dynamic">
+    @if (label()) {
+      <mat-label>{{ label() }}</mat-label>
+    }
+
+    <mat-icon matPrefix>search</mat-icon>
+
+    <input
+      matInput
+      type="text"
+      [placeholder]="placeholder()"
+      [formControl]="searchControl"
+      [matAutocomplete]="auto"
+      [attr.aria-label]="ariaLabel()"
+    />
+
+    @if (searchControl.value) {
+      <button matSuffix mat-icon-button aria-label="Clear search" (click)="clearSearch()" type="button">
+        <mat-icon>close</mat-icon>
+      </button>
+    }
+
+    <mat-autocomplete
+      #auto="matAutocomplete"
+      [displayWith]="displayWith"
+      (optionSelected)="onOptionSelected($event.option.value)"
+      [autoActiveFirstOption]="false"
+    >
+      @for (option of filteredOptions(); track trackByValue($index, option)) {
+        <mat-option [value]="option">
+          {{ option.label }}
+        </mat-option>
+      }
+
+      @if (filteredOptions().length === 0 && searchQuery()) {
+        <mat-option [disabled]="true"> No results found </mat-option>
+      }
+    </mat-autocomplete>
+  </mat-form-field>
+
+  <div class="results-counter" aria-live="polite">
```

Comment:
I don't know enough about the aria-live attribute to know if this is the correct usage. Could you expand a bit on your reasoning?